array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,18 @@
+/* exported initial */
+function initial(array) {
+  var removingLastItem = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== array.length - 1) {
```

Comment:
Instead of having this conditional in the loop, you can adjust your condition to stop sooner, and remove the need for this.